Orange/widgets/data/owcreateclass.py

```diff
@@ -255,7 +259,11 @@ def set_data(self, data):
         if not model:
             self.attribute = None
             self.Outputs.data.send(None)
+            self.info.set_input_summary(self.info.NoInput)
```

Comment:
self.data is not always None when there is no model.
In case of 'Iris' dataset on the input, the input info summary is not correct.